bundles/org.openhab.ui/web/src/components/addons/addon-card.vue

```diff
@@ -1,45 +1,56 @@
-      </div>
-      <div class="addon-card-label">
-        {{ addon.label }}
+  <f7-link v-if="addon" class="addon-card padding-right-half" :href="addon.id">
+    <div class="addon-card-inner">
+      <div class="addon-card-headline">
+        <div>{{ headline || autoHeadline || "&nbsp;" }}</div>
       </div>
-      <div v-if="addon.verifiedAuthor" class="addon-card-subtitle">
-        {{ addon.author }}
-        <f7-icon v-if="addon.verifiedAuthor" size="15" :color="$f7.data.themeOptions.dark === 'dark' ? 'white' : 'blue'" f7="checkmark_seal_fill" style="margin-top: -3px;" />
+      <div class="addon-card-title">
+        <div class="addon-card-title-after">
+          <f7-preloader v-if="addon.pending" color="blue" />
+          <f7-button v-else-if="addon.installed" class="install-button prevent-active-state-propagation" text="Remove"
+                     color="red" round small @click="buttonClicked" />
+          <f7-button v-else class="install-button prevent-active-state-propagation"
+                     :text="installActionText || 'Install'" color="blue" round small @click="buttonClicked" />
+        </div>
+        <div class="addon-card-label">
+          {{ addon.label }}
+        </div>
+        <div v-if="addon.verifiedAuthor" class="addon-card-subtitle">
+          {{ addon.author }}
+          <f7-icon v-if="addon.verifiedAuthor" size="15"
+                   :color="$f7.data.themeOptions.dark === 'dark' ? 'white' : 'blue'" f7="checkmark_seal_fill"
+                   style="margin-top: -3px;" />
+        </div>
+        <div v-else-if="addon.properties && addon.properties.views" class="addon-card-subtitle">
+          <addon-stats-line :addon="addon" :iconSize="15" />
+        </div>
       </div>
-      <div v-else-if="addon.properties && addon.properties.views" class="addon-card-subtitle">
-        <addon-stats-line :addon="addon" :iconSize="15" />
+      <div class="logo-square">
+        <f7-icon v-show="!logoLoaded" size="150" color="gray" :f7="addonIcon" class="card-default-icon" />
+        <img v-if="!logoError" class="lazy logo" :style="{ visibility: logoLoaded ? 'visible': 'hidden' }" ref="logo"
+             :data-src="imageUrl">
       </div>
     </div>
-    <div class="logo-square">
-      <f7-icon v-show="!logoLoaded" size="150" color="gray" :f7="addonIcon" class="card-default-icon" />
-      <img v-if="!logoError" class="lazy logo" :style="{ visibility: logoLoaded ? 'visible': 'hidden' }" ref="logo" :data-src="imageUrl">
-    </div>
   </f7-link>
 </template>
 
 <style lang="stylus">
 .addon-card
-  display flex
-  flex-direction column
-  align-items flex-start
-  align-content flex-end
-  color inherit
-  scroll-snap-align center center
-  padding 5px 0px 0px
+  padding 5px
+  width: 100%
   position relative
-  margin-bottom 1rem
+
+  .addon-card-inner
+    width 100%
+    height 100%
+    background-color #1c1c1d
```

Comment:
```suggestion
    background-color var(--f7-list-link-hover-bg-color)
```
And I would actually prefer it on `:hover` so that the look is the same as the list items (the color would change on the desktop theme only, no change for iOS or android)